.gitignore

```diff
@@ -1,2 +1,4 @@
 *__pycache__
 *.pytest_cache
+.coverage
+htmlcov/*
```

Comment:
should it be *.coverage and *htmlcov to remove such files in any subdirectory ?

Reply:
As far as I understand htmlcov/* will cover all folder called "htmlcov", no?

Reply:
Nops, it'll only hit root, for recursive you should start with *
![image](https://user-images.githubusercontent.com/1215497/111381059-f4278300-8683-11eb-8fce-a20657e8c20b.png)

Reply:
https://git-scm.com/docs/gitignore#_examples

I think you're putting a slash before the folder's name. Without it, it works fine with subdirectories.

Reply:
Is that clear enough ?
![image](https://user-images.githubusercontent.com/1215497/111536323-de2fc600-8748-11eb-850b-7be0f88c4bdb.png)

Reply:
the asterisk means zero or more characters, if you expand what you wrote you'll understand.  
If it's still not clear, please provide an example to show that works, because that is not the case.